Added Python2 and Swift solutions for day 11

della-90 opened the pull request.

della-90 committed:
Added Python2 and Swift solutions for day 11

tamycova commented:
I had a similar solution for python3, trying to clean it (because it looks really bad with the hardcoding of the i and j), any insights? :D

della-90 commented:
Well if you want to avoid magic numbers, then in Python 2 you can do like this:

```
#!/bin/python

import sys


arr = []
for line in sys.stdin:
    arr.append(map(int, line.strip().split()))

m = -sys.maxint-1
n = len(arr)
for i in xrange(1, n-1):
    for j in xrange(1, n-1):
        partial = arr[i][j]+arr[i-1][j-1]+arr[i-1][j]+arr[i-1][j+1]+arr[i+1][j-1]+arr[i+1][j]+arr[i+1][j+1]
        if partial > m:
            m = partial
print m
```

And in Python 3 it could be:

```
#!/bin/python3
import sys

arr = []
for line in sys.stdin:
    arr.append(list(map(int, line.strip().split())))

m = -sys.maxsize-1
n = len(arr)
for i in range(1, n-1):
    for j in range(1, n-1):
        partial = arr[i][j]+arr[i-1][j-1]+arr[i-1][j]+arr[i-1][j+1]+arr[i+1][j-1]+arr[i+1][j]+arr[i+1][j+1]
        if partial > m:
            m = partial
print(m)
```
